flink-runtime/src/main/java/org/apache/flink/runtime/operators/hash/MutableHashTable.java

```diff
@@ -57,7 +58,7 @@
  * <pre>
  * +----------------------------- Bucket x ----------------------------
  * |Partition (1 byte) | Status (1 byte) | element count (2 bytes) |
- * | next-bucket-in-chain-pointer (8 bytes) | reserved (4 bytes) |
+ * | next-bucket-in-chain-pointer (8 bytes) | bitSet (2 bytes) | reserved (2 bytes) |
```

Comment:
Can you change `bitSet` to something that describes its purpose such as for example `probeFlags`?